package.json

```diff
@@ -48,11 +48,12 @@
   },
   "scripts": {
     "test": "grunt test",
-    "prepublish": "grunt concat-core-sdk",
+    "prepublish": "npm run snyk-protect; grunt concat-core-sdk",
```

Comment:
What does `snyk-protect` do? Is it required considering that we are already running the checks for every pull request?

Reply:
The purpose is to modify the files with patches before a publish.